db/migrate/20200513190742_backup_user_assets.rb

```diff
@@ -0,0 +1,10 @@
+class BackupUserAssets < Mongoid::Migration
+  def self.up
+    UserAssetService.push_assets_to_remote
+  end
+
+  def self.down
+    bucket = UserAssetService.get_storage_bucket
+    bucket.delete
```

Comment:
hmm...  I think this might be a rare case where we don't want a true 'undo' migration.  I'm imagining a scenario where we're rolling back production because of massive errors, and having the bucket as a backup source of user data could be a lifesaver.   Essentially, if we're rolling this back, I'd want to make sure the decision to delete the bucket doesn't happen automatically

Reply:
Fair point.

Reply:
f475a86